task1/src/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import { HelloWorld } from './HelloWorld';
+import { Counter } from './Counter';
+import { SearchForm } from "./SearchForm";
+import { GenreToggle } from "./GenreToggle";
+import reportWebVitals from './reportWebVitals';
```

Comment:
use same approach for `''` or `""` everywhere. 
it can be handled by linters

Reply:
updated